CommunicationSeminar/settings/production.py

```diff
@@ -206,6 +206,8 @@ def env_get_bool(name, default=None):
     SESSION_COOKIE_SECURE = True
     CSRF_COOKIE_SECURE = True
 
+"""
+#old logging settings
```

Comment:
I reckon we can remove the old settings - if we ever cant to refer to them we can dig them out of git.

Reply:
Sounds like a plan.

Reply:
Actually could you make that a suggestion like you did above so that I can approve it and add that commit to this pull request? I am not allowed to do so as the creator of this pull request.